.github/actions/wheel/action.yml

```diff
@@ -0,0 +1,86 @@
+name: "Wheel"
+description: "Build wheels"
+inputs:
+  package_version:
+    description: 'Package version'
+    required: true
+    default: '0.1.1'
```

Comment:
Would it be better to default to some dev version? 0.0.dev5555

Reply:
Well, only if we add an extra logic to [setup.py](https://github.com/microsoft/DeepGNN/blob/main/src/python/setup.py#L136) file for dev versions to depend on each other(e.g. deepgnn-tf dev version should also depend on deepgnn-ge dev version). We save artifacts with the workflow instead of pushing them to a feed, which means we don't have to be smart :)